addons/hr_holidays/models/hr_leave_allocation.py

```diff
@@ -489,11 +497,18 @@ def _process_accrual_plans(self, date_to=False, force_period=False, log=True):
             # get current level and normal period boundaries, then set nextcall, adjusted for level transition and carryover
             # add days, trimmed if there is a maximum_leave
             while allocation.nextcall <= date_to:
+                is_leave = False
```

Comment:
Why not grouping everything using this variable (declaration, assignation and condition) to make it clearer? Also, I'm not sure that the variable name is the best; it's linked to the allocation, and it's relative to having future leaves :thinking: